.github/workflows/main.yml

```diff
@@ -160,3 +160,9 @@ jobs:
           $env:TASK = "${{ matrix.task }}"
           conda init powershell
           & "$env:GITHUB_WORKSPACE/.ci/test_windows.ps1"
+  allgood:
+    runs-on: ubuntu-latest
+    needs: [test]
+    steps:
+    - name: echo allgood
+      run: echo "allgood"
```

Comment:
Could you please make wording more intelligent please? 😄

Reply:
```suggestion
    - name: echo more intelligent please
      run: echo "more intelligent please"
```

Reply:
sorry, we mean could you make it something like

```yaml
- name: simple-task
   echo: "this is a minimal task"
```

Reply:
and instead of `allgood`, could you please name this job `all-github-actions-tests`, to make it clearer what it is there for?

and can you add a comment linking to our discussion? https://github.community/t/is-it-possible-to-require-all-github-actions-tasks-to-pass-without-enumerating-them/117957/4?u=graingert

So that in the future, when someone looks at this file, they understand why we have this extra job that seems to only run a small `echo`